back/engines/commercial/public_api/spec/acceptance/v2/users_spec.rb

```diff
@@ -5,78 +5,87 @@
 require './engines/commercial/public_api/spec/acceptance/v2/support/shared'
 
 resource 'Users' do
-  explanation 'All registrations on the citizenlab platform.'
+  explanation 'All user registrations on the platform.'
 
   include_context 'common_auth'
 
   let!(:users) { create_list(:user_with_demographics, 5) }
 
   get '/api/v2/users/' do
-    route_summary 'Get a page of users.'
-    route_description 'Endpoint to retrieve users of the platform. The most recent users are returned first. The endpoint supports pagination.'
+    route_summary 'List users'
+    route_description <<~DESC.squish
+      Retrieve a paginated list of all the users in the platform, with the most recently
+      created ones first.
+    DESC
 
     include_context 'common_list_params'
 
     # TODO: Filters - first_participated_at, status
 
-    context 'Unfiltered paged request' do
+    context 'when the page size is smaller than the total number of users' do
       let(:page_size) { 2 }
 
-      example_request 'Successful response' do
+      example_request 'List only the first page of users' do
```

Comment:
Have to check whether 'test criteria' read correctly when the open api documentation is produced - I'd focused my tests on what the api docs looked like rather than describing the tests well! Not an issue though, it can always be changed if it doesn't read so well.